src/main/java/com/playonlinux/ui/impl/javafx/installwindow/InstallWindow.java

```diff
@@ -283,7 +283,7 @@ public void update(Observable o, Object arg) {
 
     public void clearSearch() {
         searchWidget.clear();
-        availableInstallerListWidget.setSearchFilter("");
+        availableInstallerListWidget.getFilter().setTitle(null);
```

Comment:
I guess a blank title is perfectly valid here. No need to introduce a null pointer